htdocs/admin/emailcollector_list.php

```diff
@@ -594,6 +596,33 @@
 print '</table>'."\n";
 print '</div>'."\n";
 
+$formcategory = new FormCategory($db);
```

Comment:
No need of $formcategory (a specialiaed class to manage categories). 
You can use $form that is always defined

Reply:
Also fixed in https://github.com/Dolibarr/dolibarr/pull/20669/commits/bdfe7f12d90051e004dc2c9a715d0a7aeb0868c7